### PR TITLE
Add FTP deprecation, update TLS opt-out deadline

### PR DESCRIPTION
**Pull request by ericlaw1979**

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ The web is constantly evolving to improve the user experience, security, and pri
...
 | Deprecate AppCache | [Chrome+1](#release-comments) \(Edge v86\)  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, please review the [WebDev documentation][WebDevAppCacheRemoval].  The Microsoft rollout schedule for deprecation is planned for one release after Chrome.  Requesting an [AppCache OriginTrial Token][AppCacheOriginTrial] allows sites to continue to use the deprecated API until Edge v90.  |  
 | Removal of Adobe Flash | Edge v88  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, please review the [Adobe Flash Chromium Roadmap][ChromiumFlashRoadmapSupportRemoved].  | 
+| Disable and remove FTP | Edge v88  | Edge v87 | In v87, FTP support will be disabled by default. In v88, FTP support will be removed. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, please review the [Chrome Platform Status Entry][[ChromePlatformStatus6246151319715840].  |
```
Comment (Reezaali) [inline microsoft-edge/web-platform/site-impacting-changes.md:26]:
```suggestion
| Turn off and remove FTP | Edge v88  | Edge v87 | In v87, FTP support will be turned off by default. In v88, FTP support will be removed. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, please review the [Chrome Platform Status Entry][[ChromePlatformStatus6246151319715840].  | 
```

Comment (ericlaw1979) [reply microsoft-edge/web-platform/site-impacting-changes.md:26]: If we prefer "Turn off" over "Disable" for whatever reason that's fine with me, but we should probably change the other uses in the document for consistency?

Comment (Reezaali) [reply microsoft-edge/web-platform/site-impacting-changes.md:26]: Sure - turn off or remove seem to be good options for the other sentences where it was used. Can you update the other instances in the doc, or would you like me to take care of that? Thanks.

Comment (jm-trd-ms) [reply microsoft-edge/web-platform/site-impacting-changes.md]: Updated.

**Review by jm-trd-ms — APPROVED** [review]
I updated and reviewed the content, please sign-off.

**Review by ericlaw1979 — COMMENTED** [review]
LGTM

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**